compiler/rustc_error_messages/locales/en-US/interface.ftl

```diff
@@ -4,3 +4,40 @@ interface_ferris_identifier =
 
 interface_emoji_identifier =
     identifiers cannot contain emoji: `{$ident}`
+
+mixed_bin_crate =
```

Comment:
These should start with the crate name, e.g.

```suggestion
interface_mixed_bin_crate =
```

This helps us avoid conflicts between Fluent resources.

Reply:
I understand the need to namespace them, but it feels weird that being in the file `interface.ftl` doesn't already namespace them, especially since I can already refer to them via a path like `interface::mixed_bin_crate`. I think this should have been a compile error.

Reply:
There's a PR up to make this a compile error, it just hasn't landed yet.